task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -48,24 +69,32 @@ public int Pop()
             else if (count <= elements.Length / 2.0)
             {
                 Resize(elements.Length / 2);
-            }            
+            }
             return elements[--count];
         }
 
-        public void Push(int item)
+        /// <summary>
+        /// Adds a new element to the top of the Stack.
+        /// </summary>
+        /// <param name="item">Takes itam as input (increases the size of the array if it is full).</param>
+        public void Push(T item)
         {
             if (count >= elements.Length)
             {
                 Resize(elements.Length * 2);
-            }            
+            }
             elements[count++] = item;
         }
 
+        /// <summary>
+        /// This method changes the actual size of the Stack.
+        /// </summary>
+        /// <param name="newSize">Takes as input new size(new array size).</param>
```

Comment:
Please avoid array word in comments to this code. It can be confusing. There Stack class in front of us but you speak about some array everytime.